src/svelte/src/components/DataMetrics/DataMetrics.svelte

```diff
@@ -257,6 +268,16 @@ limitations under the License.
         case MessageCommand.profile:
           numAscii = msg.data.data.numAscii as number
           byteProfile = msg.data.data.byteProfile as number[]
+
+          // character count data
```

Comment:
I wonder if the MessageCommand ticket that @nlewis05 is working may effect the data extraction from the vscode websocket messages and in turn effect this portion as well.

Or to clean this up, the `CharacterCountData` class can have an extraction function that takes the message as a argument